el/src/main/java/com/sun/ts/tests/el/api/jakarta_el/compositeelresolver/ELClientIT.java

```diff
@@ -37,26 +37,46 @@
 import jakarta.el.ELContext;
 import jakarta.el.ELManager;
 
-public class ELClient extends ServiceEETest {
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+
+import java.lang.System.Logger;
+
+public class ELClientIT extends ServiceEETest {
```

Comment:
ServiceEETest has code related to harness, do we require for Arquillian + Junit runs?